client/src/components/AboutPage.js

```diff
@@ -1,11 +1,186 @@
-import React from "react";
-import Page from "./Page";
-import { Container, Row, Col } from "reactstrap";
-import { img } from "./Speaker2.css";
-import hoop from "../images/SVG/hoop.svg";
-import x from "../images/SVG/x.svg";
-import { image } from "../images";
-import ReadMore from './ReadMore'
+import React from 'react';
+import styled from 'styled-components';
+
+import Page from './Page';
+import ReadMore from './ReadMore';
+
+import x from '../images/SVG/x.svg';
+
+import mediaQueryMin from '../styles/MediaQueriesMixin';
+import {
+  AlignCenterColumn,
+  HeadingAligner,
+  HeadingHoop,
+  PageHeading,
+  BreakLineInverted,
+  Heading4,
+  Paragraph,
+  SimpleLink,
+  Heading2,
+  BreakLineMain,
+} from './GlobalStyledComponents/ReversimStyledComps';
+
+// styled-componets components
+
+const IntroContainer = styled.div`
+  ${({ theme: { color, font, space, mq } }) => `
+    padding: calc(2 * ${space.xxl}) 0 calc(3 * ${space.xl}) 0;
+
+    background-color: ${color.background_2};
+    background-image: url(${x});
+    background-repeat: no-repeat;
+    background-size: 600px;
+    background-position: 50% 120%;
+
+    color:  ${color.text_1};
+    font-size: ${font.size_md};
+  `}
+`;
+
+const IntroInnerContainer = styled(AlignCenterColumn)`
+  display: block;
+`;
+
+const MainContainer = styled(AlignCenterColumn)`
+  ${({ theme: { color } }) => `
+    background-color: ${color.background_4};
+    display: block;
+    `}
+`;
+
+const IntroHeadingContainer = styled(HeadingAligner)`
+  ${({ theme: { space } }) => `
+    padding-top: calc(3 * ${space.xl});
+    `}
+`;
+
+const IntroTextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  ${mediaQueryMin.l`
+    flex-direction: row;
+  `}
+`;
+
+const IntroParagraphContainer = styled.div`
+  ${({ theme: { space } }) => `
+    flex: 1;
+    margin-right: calc(3 * ${space.l});
+    padding-top: ${space.xxl};
+  `}
+`
+
+const IntroSubHeading = styled(Heading4)`
+  ${({ theme: { font,  } }) => `
+    font-weight: ${font.weight_medium};
+    `}
```

Comment:
sometimes the `}` is not vertically aligned to the opening one, I see it in many places, this is just an example